project/plugins.sbt

```diff
@@ -1,10 +1,10 @@
 addSbtPlugin("ch.epfl.scala"                     % "sbt-bloop"                     % "1.4.8")
 addSbtPlugin("ch.epfl.scala"                     % "sbt-scalafix"                  % "0.9.29")
+addSbtPlugin("ch.epfl.scala"                     % "sbt-version-policy"            % "1.2.1")
 addSbtPlugin("com.eed3si9n"                      % "sbt-buildinfo"                 % "0.10.0")
 addSbtPlugin("com.geirsson"                      % "sbt-ci-release"                % "1.5.7")
 addSbtPlugin("com.github.cb372"                  % "sbt-explicit-dependencies"     % "0.2.16")
 addSbtPlugin("com.thoughtworks.sbt-api-mappings" % "sbt-api-mappings"              % "3.0.0")
-addSbtPlugin("com.typesafe"                      % "sbt-mima-plugin"               % "0.8.1")
```

Comment:
sbt-version-policy already includes sbt-mima-plugin.

However, it currently depends on version 0.9.2, which is known to suffer from https://github.com/lightbend/mima/issues/639 (which makes this PR fail). We will probably have to wait for 0.9.3…